package.json

```diff
@@ -1,12 +1,14 @@
 {
     "name": "@brightlayer-ui/colors",
-    "version": "4.0.0-alpha.1",
+    "version": "4.0.0-alpha.2",
     "description": "Brightlayer UI colors for Eaton applications",
-    "main": "./dist/commonjs/index.js",
-    "types": "./dist/commonjs/index.d.ts",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
     "scripts": {
         "test": "./buildTest.sh",
-        "build": "yarn && tsc --module commonjs --outDir dist/commonjs && tsc --module ES2015 --outDir dist/es2015",
+        "build": "yarn tsc",
```

Comment:
If you are removing the inline configuration options and only creating a single output format, then I think it should be commonjs instead of ES2015 (you'll need to update tsconfig.json).